src/Api/Checkout.php

```diff
@@ -0,0 +1,162 @@
+<?php namespace Lean\Woocommerce\Api;
+
+use Lean\AbstractEndpoint;
+use Epoch2\HttpCodes;
+use Lean\Woocommerce\Utils\ErrorCodes;
+use Lean\Woocommerce\Utils\Hooks;
+
+/**
+ * Class Checkout.
+ *
+ * @package Lean\Woocommerce\Api
+ */
+class Checkout extends AbstractEndpoint
+{
```

Comment:
Thank you, I do have a personal preference, I prefer like this, on the same line as well. I agree about consistency also! Thank you